drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -942,13 +942,15 @@ function DragAndDropBlock(runtime, element, configuration) {
         }
     };
 
-    var sliderInitializer = function() {
-        itemSlider = $root.find('.item-bank').bxSlider({
-            pager: false,
-            infiniteLoop: false,
-            startSlide: currentSlideIndex,
-            speed: 0
-        });
+    var initializeSlider = function() {
+        if (pageLoaded) {
```

Comment:
Thanks for reinstating this @jawad-khan -- could you please add a comment explaining why this is needed, so it doesn't get removed in the future?  e.g.
```javascript
// Need to wait for the full page load before loading the bxSlider,
// so that images contained in the droppable items are loaded properly.
```